include/client/networkclient.h

```diff
@@ -135,3 +135,5 @@ public slots:
 
 }  // namespace client
 }  // namespace veles
+
+Q_DECLARE_METATYPE(veles::client::NetworkClient::ConnectionStatus)
```

Comment:
To do it  "by the books" you should probably also add `qRegisterMetaType` call for this type (it seems to work now without it, but I'd like to avoid situation where on one specific version of Qt it doesn't work) - 

> Note that if you intend to use the type in queued signal and slot connections or in QObject's property system, you also have to call qRegisterMetaType() since the names are resolved at runtime.